contrib/upgrade.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+# this is an upgrade script, for gitea deployed on linux as systemd service
+# depends on: curl, xz, sha256sum, gpg
+# usage:      upgrade.sh [version]
+
+# change the variables below for your local setup
+giteaversion=${1:-1.15.10}
+giteabin=/usr/local/bin/gitea
+giteahome=/var/lib/gitea
+giteaconf=/etc/gitea/app.ini
+giteauser="git"
+giteacmd="sudo -u $giteauser $giteabin -c $giteaconf -w $giteahome"
+
+# download new binary
+binname=gitea-${giteaversion}-linux-amd64
+binurl="https://dl.gitea.io/gitea/${giteaversion}/${binname}.xz"
+echo downloading $binurl
+cd $giteahome # needed for gitea dump later
+curl -sSfL "$binurl" > ${binname}.xz
```

Comment:
I think we should prefer using `-o "${binname}.xz"` instead of using the `>`.  Ditto that for the 2 other curl commands as well.

Reply:
Also, we could simplify it to 1 command by using `curl`'s globbing abilities.